apiserver/params/params.go

```diff
@@ -498,6 +498,12 @@ type DestroyApplicationParams struct {
 	DestroyStorage bool `json:"destroy-storage,omitempty"`
 }
 
+// DestroyConsumedApplicationsParams holds bulk parameters for the
+// Application.DestroyConsumedApplication call.
+type DestroyConsumedApplicationsParams struct {
+	ApplicationTags []string `json:"application-tags"`
```

Comment:
please structure this like:

```go
type DestroyConsumedApplicationsParams struct {
    Applications []DestroyConsumedApplicationParams `json:"applications"`
}

type DestroyConsumedApplicationParams struct {
    ApplicationTag string `json:"application-tag"`
}
```

so we can add application-specific flags as necessary